crates/client-api/src/routes/database.rs

```diff
@@ -222,7 +222,7 @@ impl<'a> EntityDef<'a> {
     }
 }
 
-fn entity_description_json(description: WithTypespace<EntityDef>, expand: bool) -> Option<Value> {
+fn entity_description_json(description: WithTypespace<EntityDef>) -> Option<Value> {
```

Comment:
[Review this with whitespace hidden](https://github.com/clockworklabs/SpacetimeDB/pull/1755/files?diff=unified&w=1#diff-728f8c9320d820b1deaf6965ec3fffb94ef62e0c33005ce8a45cef7f0d8c2fd1)